core/roboconf-messaging-rabbitmq/src/main/java/net/roboconf/messaging/rabbitmq/internal/RabbitMqClient.java

```diff
@@ -71,13 +72,21 @@
 	 */
 	static final String DEFAULT_PASSWORD = "guest";
 
+
 	protected final Logger logger = Logger.getLogger(this.getClass().getName());
 	protected final String messageServerIp, messageServerUsername, messageServerPassword;
+	private final WeakReference<ReconfigurableClient<?>> reconfigurable;
 	protected LinkedBlockingQueue<Message> messageQueue;
 	Channel channel;
-	private final WeakReference<ReconfigurableClient<?>> reconfigurable;
 
 
+	/**
+	 * Constructor.
```

Comment:
This is just noise! I did remove that because that kind of comment is just as useful as a misconfigured code style checker ;) Maybe parameter description could calm me down... `</joking>`